### ISSUE TITLE
Extending the application to other robots

### ISSUE THREAD

**Issue by programmeddeath1** [facebookresearch/vjepa2#26]
Hi thank you for this amazing paper! seems like a great leap forward for physical intelligence.

I want to try and understand how this can be extended to run reach/pick and place for other robots like below 

1) 6 axis robots with different size and kinematics.
2) 3-axis Delta robots
3) Gantry robots with x, y, z - position A, B, C  orientation/grasp movements.

I saw in the paper that Action Conditioning World Model training was done on the droid dataset with 7D vectors defining the end-effector state relative to the base of the robot for each image. For extending to other robots as above how will this training process change and how much of the existing original model be used as is?

**Comment by quxianjiuguo** [comment]
@programmeddeath1 
Hello, I would like to use this project on my own ur5e robotic arm to complete fine-tuning, prediction, etc. What data format do I need to prepare?

**Comment by quxianjiuguo** [comment]
@programmeddeath1 
Hello, I would like to use this project on my own ur5e robotic arm to complete fine-tuning, prediction, etc. What data format do I need to prepare?

**Comment by ballasnicolas** [comment]
Hi, we only tried the approach with Franka robots in our work so far, but the same methodology should be applicable to other embodiment. You would need to learn the VJEPA2 AC predictor using your own robotic data. Please refer to https://github.com/facebookresearch/vjepa2/blob/main/app/vjepa_droid/train.py for the VJEPA 2 AC training script.
